src/main/java/com/deque/axe/android/AxePropCalculator.java

```diff
@@ -0,0 +1,192 @@
+package com.deque.axe.android;
+
+import com.deque.axe.android.constants.AndroidClassNames;
+import com.deque.axe.android.utils.AxeTextUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class AxePropCalculator {
+
+  private static final String SPACE = " ";
+  private static Map<String, String> propMap = new HashMap<>();
+  private final String text;
+  private final String contentDescription;
+  private final String labelText;
+  private final String value;
+  private final boolean isEnabled;
+  private final String className;
+  private final String hintText;
+
+  private String calculatedProp;
+
+  AxePropCalculator(String text,
```

Comment:
This might just be a difference between Android and iOS, but for iOS, instead of passing in each individual property that I need to calculate the properties, I ended up just passing in the entire `AxeView` object.  Then, I was able to grab the properties that I needed for each CalculatedProp.

Reply:
Ok, I am trying to think about advantage of getting children while we are calculating props.